src/mixins/_media-query.scss

```diff
@@ -72,6 +95,34 @@ $last-breakpoint: list.nth($breakpoint-keys, -1);
   }
 }
 
+/// Repeat a chunk of CSS for multiple standard breakpoints. Useful for keeping
+/// responsive CSS nice and DRY.
+///
+/// @param {String} $from [$first-breakpoint] - The first/narrowest breakpoint
+/// token key to start looping over.
+/// @param {String} $to [$last-breakpoint] - The last/widest breakpoint token
+/// key to finish looping over.
+/// @param {Bool} $include-default [true] - If true, the `@content` will be
+/// output in the current parent before any media queries.
+/// @content
+/// @output The `@content` nested within variations of the parent class that
+/// include media query token segments.
+@mixin breakpoint-classes-max-width(
+  $from: $first-breakpoint,
+  $to: $last-breakpoint,
+  $include-default: true
+) {
+  @if ($include-default) {
+    @content;
+  }
+
+  @include breakpoint-loop-max-width($from, $to) using ($key) {
+    &\@#{$key} {
```

Comment:
Should it use different classes here? It might be confusing for `@s` to mean "below s breakpoint" in some contexts and "above s breakpoint" in other contexts.

What do you think about using `@below-s` or `@max-s`. Also we could potentially rename the other ones too to `@above-s` or `@min-s` for clarity.

Reply:
I don't think we should rename the `min-width` ones. Since we design for mobile first, it's usually more common for the mobile to be the default and for changes to apply at larger sizes.

I'm re-reading some other CSS frameworks' reasoning behind only having `min-width` utilities ([example](https://tailwindcss.com/docs/responsive-design)) and I'm second-guessing this direction. Is it possible to carve off the `c-button-group` changes as a separate issue for now?

Reply:
> Is it possible to carve off the c-button-group changes as a separate issue for now?

Sure, I can open an issue. What should happen to this PR? Should I revert it back to not use the button-group pattern?

Reply:
Or you could just have it use the button group pattern in its default state. Either way is fine.